viewing_party/party.py

```diff
@@ -0,0 +1,125 @@
+def create_movie(movie_title, genre, rating):
+    if movie_title is None:
+        return None
+    if genre is None:
+        return None
+    if rating is None:
+        return None
+    movie = {}
+    movie["title"] = movie_title
+    movie["genre"] = genre
+    movie["rating"] = rating
```

Comment:
we can also do this all at once, too!

```suggestion
    movie = {"title": title, "genre": genre, "rating": rating}
```